src/components/CheckboxInput/CheckboxInput.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useState } from "react";
+import { useFormContext } from "react-hook-form";
+import check from "../../images/check.svg";
+import { FormValues } from "../Form/Form";
+
+interface CheckboxInputProps {
+  name: keyof FormValues;
+  label: string;
+  required?: boolean; // Optional prop for making the checkbox required
+  requiredError?: string; // Custom error message for the required validation
+  error?: { message: string };
+}
+
+const CheckboxInput: React.FC<CheckboxInputProps> = ({
+  name,
+  label,
+  required = false, // Default is not required
+  requiredError = "You must agree to the terms", // Default error message
```

Comment:
I believe this would look better if it were aligned with the rest of the check box input text. 
<img width="780" alt="image" src="https://github.com/user-attachments/assets/a6b29bcf-6f8e-493b-9031-e32599cfea93">